benchmarking/benchmarking.go

```diff
@@ -0,0 +1,85 @@
+package benchmarking
```

Comment:
Maybe we should place it in `cmd/` to better conform to the Go conventions.

Reply:
At this point this is not a binary, it just provides extra cmd flags for go test cmd. But I agree, once we start getting closer to substrate compatibility this will need to be turned into a cmd binary